test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -367,39 +353,96 @@ func TestAPIExportPermissionClaims(t *testing.T) {
 	wildwestVCClients, err := wildwestclientset.NewForConfig(apiExportVWCfg)
 	require.NoError(t, err)
 
-	t.Logf("verify that the export resource is retrievable")
+	t.Logf("verify that the exported resource is retrievable")
 	cowboysProjected, err := wildwestVCClients.WildwestV1alpha1().Cowboys("").List(logicalcluster.WithCluster(ctx, logicalcluster.Wildcard), metav1.ListOptions{})
 	require.NoError(t, err)
 	require.Equal(t, 1, len(cowboysProjected.Items))
 
-	t.Logf("Ensuring the appropriate core/v1 resources are available")
 	dynamicVWClusterClient, err := kcpdynamic.NewClusterDynamicClientForConfig(apiExportVWCfg)
 	require.NoError(t, err, "error creating dynamic cluster client for %q", apiExportVWCfg.Host)
 
-	grantedGVRs := []schema.GroupVersionResource{
+	sheriffsGVR := schema.GroupVersionResource{Version: "v1", Resource: "sheriffs", Group: "wild.wild.west"}
+
+	claimedGVRs := []schema.GroupVersionResource{
 		{Version: "v1", Resource: "configmaps"},
 		{Version: "v1", Resource: "secrets"},
 		{Version: "v1", Resource: "serviceaccounts"},
 		{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "clusterroles"},
 		{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "clusterrolebindings"},
+		sheriffsGVR,
 	}
 
-	for _, gvr := range grantedGVRs {
+	t.Logf("verify that we get empty lists for all claimed resources because the claims have not been accepted yet")
+	for _, gvr := range claimedGVRs {
 		t.Logf("Trying to wildcard list %q", gvr)
 		list, err := dynamicVWClusterClient.Cluster(logicalcluster.Wildcard).Resource(gvr).List(ctx, metav1.ListOptions{})
 		require.NoError(t, err, "error listing %q", gvr)
+		require.Empty(t, list.Items, "expected 0 items but got %#v", list.Items)
+	}
+
+	t.Logf("retrieving cowboys apibinding")
+	apiBinding, err := kcpClusterClient.ApisV1alpha1().APIBindings().Get(logicalcluster.WithCluster(ctx, consumerWorkspace), "cowboys", metav1.GetOptions{})
+	require.NoError(t, err, "error getting cowboys apibinding")
+
+	bindingWithClaims := apiBinding.DeepCopy()
+	for i := range apiBinding.Status.ExportPermissionClaims {
+		claim := apiBinding.Status.ExportPermissionClaims[i]
+		bindingWithClaims.Spec.PermissionClaims = append(bindingWithClaims.Spec.PermissionClaims, apisv1alpha1.AcceptablePermissionClaim{
+			PermissionClaim: claim,
+			State:           apisv1alpha1.ClaimAccepted,
+		})
+	}
+
+	oldJSON := encodeJSON(t, apiBinding)
+	newJSON := encodeJSON(t, bindingWithClaims)
+	patchBytes, err := jsonpatch.CreateMergePatch(oldJSON, newJSON)
+	require.NoError(t, err, "error creating patch")
+	t.Logf("patching apibinding to accept all permissionclaims")
+	apiBinding, err = kcpClusterClient.ApisV1alpha1().APIBindings().Patch(logicalcluster.WithCluster(ctx, consumerWorkspace), apiBinding.Name, types.MergePatchType, patchBytes, metav1.PatchOptions{})
+	require.NoError(t, err, "error patching apibinding to accept all claims")
```

Comment:
can you elaborate on why Patch vs. Update (as godoc)?

Reply:
Will switch to update

Reply:
Actually, going to keep patch to avoid conflicts